juan-pablo-hernandez/tarea-3/spongebobMeme.js

```diff
@@ -0,0 +1,22 @@
+let test = 'stop Making spongebob Memes!';
+
+function spongeMeme(sentence) {
```

Comment:
@camosu , find a way to make this a one-line solution using modern approaches for array operation , ex. forEach, map, reduce